lib/hanami/view/application_context.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Hanami
+  class View
+    module ApplicationContext
+      def initialize(**options)
+        @inflector = options.fetch(:inflector) { Hanami.application.inflector }
```

Comment:
@jodosha You're right, that approach wasn't necessary! I think I did it to be consistent with the params signature of the base `Context#initialize`:

```ruby
def initialize(render_env: nil, **options)
```

But this totally works and creates a more explicit signature:

```ruby
def initialize(inflector: Hanami.application.inflector, **options)
  @inflector = inflector
  super
end
```

I'm pushing that change up now.

BTW, we actually need to pass _all_ kwargs up to the base initializer so it can keep them all in `@_options` and restore them when we prepare the view context for a given render environment:

```
# From View::Context:

      # @api private
      def for_render_env(render_env)
        return self if render_env == _render_env

        self.class.new(**_options.merge(render_env: render_env))
      end
```

This is why we need to call `super` (which ensures the inflector is passed up) instead of `super(**options)` which will not pass the inflector.